packages/core/src/controllers/OptionsController.ts

```diff
@@ -19,6 +19,9 @@ export interface OptionsControllerState {
   enableAnalytics?: boolean
   metadata?: Metadata
   enableOnramp?: boolean
+  siweSession?: {
+    accounts: string[]
+  }
```

Comment:
Not sure if this should live in core.
@glitch-txs recently moved anything related to SIWE into its own package.